Library/Homebrew/test/compiler_selector_spec.rb

```diff
@@ -29,7 +29,7 @@
   describe "#compiler" do
     it "raises an error if no matching compiler can be found" do
       software_spec.fails_with(:clang)
-      software_spec.fails_with(:gcc)
+      software_spec.fails_with(:gcc_4_2)
```

Comment:
I'm not sure about this; this will change the mean for existing formula that were using `fails_with :gcc`, no?